src/@dvcorg/gatsby-theme-iterative/components/LayoutFooter/index.tsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react'
 import cn from 'classnames'
 
-import LayoutWidthContainer from 'gatsby-theme-iterative-docs/src/components/LayoutWidthContainer'
-import Link from 'gatsby-theme-iterative-docs/src/components/Link'
+import LayoutWidthContainer from '@dvcorg/gatsby-theme-iterative/src/components/LayoutWidthContainer'
+import Link from '@dvcorg/gatsby-theme-iterative/src/components/Link'
 import SocialIcon, {
   ISocialIcon
-} from 'gatsby-theme-iterative-docs/src/components/SocialIcon'
-import ShowOnly from 'gatsby-theme-iterative-docs/src/components/ShowOnly'
-import { getFirstPage } from 'gatsby-theme-iterative-docs/src/utils/shared/sidebar'
+} from '@dvcorg/gatsby-theme-iterative/src/components/SocialIcon'
+import ShowOnly from '@dvcorg/gatsby-theme-iterative/src/components/ShowOnly'
+import { getFirstPage } from '@dvcorg/gatsby-theme-iterative/src/utils/shared/sidebar'
 
-import { ReactComponent as LogoSVG } from '../../../static/img/dvc_icon-color--square_vector.svg'
-import { ReactComponent as GithubSVG } from 'gatsby-theme-iterative-docs/src/components/SocialIcon/github.svg'
-import { ReactComponent as TwitterSVG } from 'gatsby-theme-iterative-docs/src/components/SocialIcon/twitter.svg'
-import { ReactComponent as DiscordSVG } from 'gatsby-theme-iterative-docs/src/components/SocialIcon/discord.svg'
-import { ReactComponent as CmlSVG } from '../../../static/img/cml_icon-color--square_vector.svg'
-import { ReactComponent as StudioSVG } from '../../../static/img/studio_icon-color--square_vector.svg'
-import { ReactComponent as IterativeSVG } from '../../../static/img/iterative_icon-color--square_vector.svg'
-import { ReactComponent as MlemSVG } from '../../../static/img/mlem-icon.svg'
+import { ReactComponent as LogoSVG } from '../../../../../static/img/dvc_icon-color--square_vector.svg'
+import { ReactComponent as GithubSVG } from '@dvcorg/gatsby-theme-iterative/src/components/SocialIcon/github.svg'
+import { ReactComponent as TwitterSVG } from '@dvcorg/gatsby-theme-iterative/src/components/SocialIcon/twitter.svg'
+import { ReactComponent as DiscordSVG } from '@dvcorg/gatsby-theme-iterative/src/components/SocialIcon/discord.svg'
+import { ReactComponent as CmlSVG } from '../../../../../static/img/cml_icon-color--square_vector.svg'
+import { ReactComponent as StudioSVG } from '../../../../../static/img/studio_icon-color--square_vector.svg'
+import { ReactComponent as IterativeSVG } from '../../../../../static/img/iterative_icon-color--square_vector.svg'
+import { ReactComponent as MlemSVG } from '../../../../../static/img/mlem-icon.svg'
```

Comment:
Tons of things needed to be bumped two directories up because of the theme shadowing dir